grc/gui/ParamWidgets.py

```diff
@@ -32,7 +32,7 @@ def is_dark_theme(theme_name):
     if prefer_dark in ('1', 'yes', 'true', 'on'):
         theme_name = config.get(
```

Comment:
If this if statement always returns true, then finding the theme name should not be necessary - this doesn't seem right as this theme name is not used anywhere else in the code.